### ISSUE TITLE
Remove deprecated fields in hits.files and hits.fileTypeSummaries

### ISSUE THREAD

**Issue by danielsotirhos** [DataBiosphere/azul#3180]
Followup of #3171 and #3111

Remove the following deprecated fields:
- `hits[].files[].matrix_cell_count` (superseded by `matrixCellCount`)
- `hits[].files[].source` (superseded by `fileSource`)
- `hits[].fileTypeSummaries[].fileType` (superseded by `format`)
- `hits[].fileTypeSummaries[].source` (superseded by `fileSource`)
- `hits[].projects[].contributorMatrices...matrix_cell_count` (superseded by `matrixCellCount`)
- `hits[].projects[].contributorMatrices...source` (superseded by `fileSource`)
- `hits[].projects[].matrices...matrix_cell_count` (superseded by `matrixCellCount`)
- `hits[].projects[].matrices...source` (superseded by `fileSource`)

**Comment by danielsotirhos** [comment]
Blocked by https://github.com/HumanCellAtlas/data-browser/issues/1898

**Comment by hannes-ucsc** [comment]
For demo, show that the fields are gone.
